lib/licenses.nix

```diff
@@ -650,11 +650,6 @@ lib.mapAttrs mkLicense (
       fullName = "GNU General Public License v2.0 only";
     };
 
-    gpl2UBDLPlus = {
-      fullName = "GNU General Public License v3.0 or later (with UBDL exception)";
```

Comment:
I assume this saying "v3.0" was a typo?

Reply:
From looking at the package where it was used yes